include/oneapi/dpl/pstl/ranges/nanorange.hpp

```diff
@@ -527,8 +527,7 @@ namespace detail
 {
 
```

Comment:
Sorry, I think I suggested this be moved from #1174  here, but now I realize that it should just be omitted because it is within `nanorange.hpp` which I think we want to leave without changes.

Reply:
@danhoeflinger could you please resolve this conversation?